index.jsx

```diff
@@ -459,20 +465,27 @@ class Controls extends Component {
       plusBtnContents,
       minusBtnContents,
       btnClass,
+      plusBtnClass,
+      minusBtnClass,
+      controlsClass,
       scale,
       minScale,
       maxScale
     } = this.props;
 
-    const btnStyle = btnClass ? undefined : { width: 30, paddingTop: 5, marginBottom: 5 };
+    const btnStyle = { width: 30, paddingTop: 5, marginBottom: 5 };
+    const controlsStyle = controlsClass ? undefined : { position: 'absolute', right: 10, top: 10 };
 
     return (
-      <div style={{ position: 'absolute', right: 10, top: 10 }}>
+      <div style={controlsStyle} className={controlsClass}>
         <div>
           <button
             ref={(node) => { this.plusNode = node; }}
-            className={btnClass}
-            style={btnStyle}
+            className={[
```

Comment:
I would use this package: https://www.npmjs.com/package/classnames

Then your cold will look something like this and you won't have to do the `.join` or ternaries.
```
import classnames from 'classnames'
...
plusClasses = classnames(btnClass, plusBtnClass);
minusClasses = classnames(btnClass, minusBtnClass);
```

The classnames package takes care of figuring out falsy/truthy values and properly formatting the resulting class list.

Reply:
As I mentioned in my previous comment, your plugin is unique in a way that it has no dependencies and that's why I avoided using the `classnames` package. Adding dependency just because of having the logic of merging classes a bit more pretty adds unnecessary complexity and other developers first have to read the `classnames` docs in order to use it. On the other hand, if we use `.join(' ')` approach, everybody knows how to use it, it's just a native thing to do. The problem with today's devs is that they wanna use package for every tiny thing and it's sometimes helpful, but sometimes it's an overkill (and I believe in this case it is really an overkill). Don't get me wrong though, I understand your point @scottyantipa, in fact, let me tell you a story. We were refactoring quite large project into React few months ago and I decided to use `classnames` package. But during the review, one of our devs pointed out that this is a total overkill and it's better to go with `join(' ')` technique that everyone knows. At first I didn't want to accept that opinion because I had already refactored some part of code with this package, but after some time I realised that he was actually right and it adds unnecessary complexity and just another package to already long list of dependencies for a nicer sugar. Then another developer came and wanted to use `join(' ')` as well. So it's quite common technique that old-school JS devs use.

In any case, it's your project after all so if you want to use `classnames` package, I'll refactor it for sure :) I just wanted you to know my reasoning behind using `join`. Let me know your decision please so I can finalise the PR.

Reply:
> that's why I avoided using the classnames
Fair enough. However I would suggest not repeating the logic twice. I would instead add your own function `classnames` that takes care of doing the `.join` and the empty string default logic.

Reply:
Accepting, and I'll leave the above up to you!